### DISCUSSION TITLE
Spacebar, page up/down key mapping

### DISCUSSION THREAD

**Discussion by somewonderfulguy** [Valkirie/HandheldCompanion#753] [Q&A] [answered]
Could anyone please help me with mapping keyboard keys? I'm setting up layout for desktop mode and cannot find such keys as `spacebar` (`whitespace`), `page up`, `page down`.

I'm speaking of these settings:
![image](https://github.com/Valkirie/HandheldCompanion/assets/23312484/656b9657-a609-4b28-b825-0a0a884aa623)

**Comment by CasperH2O** [comment, marked as answer]
It's being worked on to improve.

![image](https://github.com/Valkirie/HandheldCompanion/assets/14330834/3c6ec247-bd6f-4147-bd40-2224e967f4d8)

**Reply by somewonderfulguy** [reply]
Oh, it's seems like I should check for updates regularly. Thanks!

**Comment by CasperH2O** [comment]
Conversion table: 
![image](https://github.com/Valkirie/HandheldCompanion/assets/14330834/3a774720-008b-4447-abb6-20f8b4d0c8ab)

**Comment by Havner** [comment]
> Could anyone please help me with mapping keyboard keys? I'm setting up layout for desktop mode and cannot find such keys as `spacebar` (`whitespace`), `page up`, `page down`.
> 
> I'm speaking of these settings: ![image](https://user-images.githubusercontent.com/23312484/258603728-656b9657-a609-4b28-b825-0a0a884aa623.png)

Currently Spacebar is an entry without a label (literally a whitespace). Page up / down  are Prior/Next.
